src/scss/_typography.scss

```diff
@@ -1,3 +1,282 @@
+html {
+  font-size: 18px;
+}
+body {
+  font-family: "Lato", sans-serif;
+  font-size: 100%;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  line-height: 1.35;
+}
+h1, h2, h3, h4, h5, h6 {
+  color: rgba(4, 85, 148, 1);
```

Comment:
Please convert colors to use variables. You may have to add them to variables.scss.